new_lineage_panel/src/ActionWidget.tsx

```diff
@@ -0,0 +1,392 @@
+import {
+  Dispatch,
+  FC,
+  FunctionComponent,
+  PropsWithChildren,
+  SetStateAction,
+  useContext,
+  useEffect,
+  useState,
+} from "react";
+import "reactflow/dist/style.css";
+import { Button, Card, CardBody, Input, Label, Tooltip } from "reactstrap";
+import AlertCircleIcon from "./assets/icons/alert-circle.svg?react";
+import ResetIcon from "./assets/icons/reset.svg?react";
+import HelpIcon from "./assets/icons/help.svg?react";
+import FeedbackIcon from "./assets/icons/feedback.svg?react";
+import ExpandIcon from "./assets/icons/expand.svg?react";
+import ArrowLeftDoubleIcon from "./assets/icons/arrow-left-double.svg?react";
+import ArrowRightDoubleIcon from "./assets/icons/arrow-right-double.svg?react";
+import ArrowLeftIcon from "./assets/icons/arrow-left.svg?react";
+import ArrowRightIcon from "./assets/icons/arrow-right.svg?react";
+import styles from "./styles.module.scss";
+import { HELP_SIDEBAR } from "./constants";
+import { init, openURL, setLegacyLineageView, CLL } from "./service_utils";
+import { LineageContext, aiEnabled } from "./App";
+import { useReactFlow } from "reactflow";
+import {
+  calculateMinLevel,
+  expandTableLineageLevelWise,
+  highlightTableConnections,
+  layoutElementsOnCanvas,
+} from "./graph";
+import classNames from "classnames";
+import { BetterPopover } from "./components/Modal/BetterPopover";
```

Comment:
try to use constant directory naming pattern. `components` start with lowercase, but Modal with uppercase